recipes/sshpass/meta.yaml

```
@@ -0,0 +1,38 @@
+{% set name = "sshpass" %}
+{% set version = "1.06" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://downloads.sourceforge.net/project/{{ name }}/{{ name }}/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c6324fcee608b99a58f9870157dfa754837f8c48be3df0f5e2f3accf145dee60
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - make
+    - pkgconfig
+
+test:
+  commands:
+    - test -x ${PREFIX}/bin/sshpass
+    - ${PREFIX}/bin/sshpass -h
```

No need for the prefix as it should be on `$PATH`
```suggestion
    - sshpass -h
```

---

You're right, fixed.